src/cluster/operator_update_workflow.go

```diff
@@ -0,0 +1,128 @@
+// Copyright © 2019 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cluster
+
+import (
+	"context"
+	"time"
+
+	"emperror.dev/errors"
+	"go.uber.org/cadence/workflow"
+	"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/typed/apiextensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+
+	"github.com/banzaicloud/pipeline/pkg/cluster"
+	"github.com/banzaicloud/pipeline/pkg/k8sclient"
+)
+
+const (
+	IntServiceOperatorUpdaterActivityName = "integrated-service-operator-updater"
+	IntServiceCRDName                     = "serviceinstances.integrated-service.banzaicloud.io"
+)
+
+type IntServiceOperatorUpdaterWorkflow struct {
+	manager *Manager
+}
+
+func NewIntServiceOperatorUpdaterWorkflow(manager *Manager) *IntServiceOperatorUpdaterWorkflow {
+	return &IntServiceOperatorUpdaterWorkflow{
+		manager: manager,
+	}
+}
+
+type IntServiceOperatorUpdaterActivityInput struct {
+	ClusterID uint
+}
+
+func (w IntServiceOperatorUpdaterWorkflow) Execute(ctx workflow.Context, input IntServiceOperatorUpdaterActivityInput) error {
+	logger := workflow.GetLogger(ctx).Sugar().With(
+		"clusterID", input.ClusterID,
+	)
+
+	commonCluster, err := w.manager.GetClusterByIDOnly(context.Background(), input.ClusterID)
+	if err != nil {
+		logger.Errorf("failed to get cluster from database: %s", err.Error())
+		return err
+	}
```

Comment:
We discussed this, so just a reminder that we need the check if the cluster exists and result cleanly in case it's gone.

In case there's an error we don't want to quit the loop.

Reply:
I fixed this